index.html

```diff
@@ -10,6 +10,56 @@
     <title>Project 0 | Hamzah*Sarah*Baraa</title>
 </head>
 <body>
+    <section id="contact-section" class="section">
+        <div class="center">
+            <div class="top">
+                <h1 class="section-title">CONTACT</h1>
+                <p class="section-description">Magnam dolores commodi suscipit. Necessitatibus eius consequatur 
+                        ex aliquid fuga eum quidem. Sit sint consectetur velit. Quisquam quos quisquam cupiditate. 
+                        Et nemo qui impedit suscipit alias ea. Quia fugiat sit in iste officiis commodi quidem hic quas.</p>        
+            </div>
+            <div class="bottom">
+                <div class="left-section">
+
+                    <div class="contact-method our-address">
+                        <i class="fas fa-map-marker-alt fa-2x"></i>
+                        <h3>Our Address</h3>
+                        <p>A108 Adam Street, New York, NY 535022</p>
+                    </div>
     
+                    <div class="email-call">
+    
+                        <div class="contact-method email-us">
+                            <i class="far fa-envelope fa-2x"></i>
+                            <h3>Email Us</h3>
+                            <p>info@example.com</p>
+                            <p>contact@example.com</p>
+                        </div>
+    
+                        <div class="contact-method call-us">
+                            <i class="fas fa-phone-alt fa-2x"></i>
+                            <h3>Call Us</h3>
+                            <p>+1 5589 55488 55</p>
+                            <p>+1 6678 254445 41</p>
+                        </div>
+    
+                    </div>
+    
+                </div>
+                
+                <div class="right-section">
+                    <form class="contact-method">
+                        <div class="input-method name-email">
+                            <input  type="text" placeholder="Your Name">
+                            <input  type="text" placeholder="Your Email">
+                        </div>
+                        <input class="mb-1 input-method" type="text" placeholder="Subject">
+                        <textarea class="mb-1 input-method" cols="30" rows="8" placeholder="Message"></textarea>
+                        <button>Send Message</button>
```

Comment:
you can add `type="submit"` to the form button 
it defines the button that submits the form data 
(more accessible, and you will need it when you want to store your data into a form-handler)

Reply:
Again, I don't care about submitting the form, or else I would've added both action and method attributes to the form itself.